server/graphql/preference/src/upsert.rs

```diff
@@ -2,12 +2,18 @@ use async_graphql::*;
 use graphql_core::{standard_graphql_error::validate_auth, ContextExt};
 use service::{
     auth::{Resource, ResourceAccessRequest},
-    preference::UpsertPreferences,
+    preference::{StorePrefUpdate, UpsertPreferences},
 };
 
+#[derive(InputObject)]
+pub struct BoolStorePrefInput {
+    pub store_id: String,
+    pub value: bool,
+}
 #[derive(InputObject)]
 pub struct UpsertPreferencesInput {
     pub show_contact_tracing: Option<bool>,
+    pub display_vaccine_in_doses: Option<Vec<BoolStorePrefInput>>,
```

Comment:
Per above, store pref upsert inputs are vecs, so can be applied to many stores at once